### PR TITLE
Add py_stringmatching

### PR DESCRIPTION
**Pull request by AbdealiJK**
Fixes https://github.com/conda-forge/staged-recipes/issues/4445

### CODE REVIEW

File: recipes/py_stringmatching/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 64794c179cefece8f337c9f458dfdce937b08efabad220e2dcdad81be3e95a66
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record=record.txt
+
+requirements:
+  build:
```
Comment (synapticarbors) [inline recipes/py_stringmatching/meta.yaml]: Please add `python` and `toolchain` as `build` dependencies.

File: recipes/py_stringmatching/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+    - setuptools
+    - numpy >=1.7.0
```
Comment (synapticarbors) [inline recipes/py_stringmatching/meta.yaml]: Please see the following about building packages that link against `numpy`:

http://conda-forge.readthedocs.io/en/latest/meta.html#building-against-numpy

File: recipes/py_stringmatching/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+    - cython
+
+  run:
```
Comment (synapticarbors) [inline recipes/py_stringmatching/meta.yaml]: Please add `python` as a `run` dependency.

File: recipes/py_stringmatching/meta.yaml [hunk 4]
```diff
@@ -0,0 +1,37 @@
...
+    - numpy >=1.7.0
+    - six
+
+test:
+  imports:
+    - "{{ name }}"
```
Comment (synapticarbors) [inline recipes/py_stringmatching/meta.yaml]: Please don't use jinja template variables in the `imports` section. Just explicitly list the module(s)

File: recipes/py_stringmatching/meta.yaml [hunk 5]
```diff
@@ -0,0 +1,37 @@
...
+
+about:
+  home: https://github.com/anhaidgroup/py_stringmatching
+  license: BSD
```
Comment (synapticarbors) [inline recipes/py_stringmatching/meta.yaml]:
```
license: BSD-3-Clause
license_family: BSD
license_file: LICENSE
```

**Comment by jakirkham** [comment]
Superceded by PR ( https://github.com/conda-forge/staged-recipes/pull/5733 ).

**Pull request closed without merging**